app-arch/atool/atool-0.39.0.recipe

```diff
@@ -0,0 +1,63 @@
+SUMMARY="A script for managing file archives"
+DESCRIPTION="The main command is aunpack which extracts files from an archive. \
+Did you ever extract files from an archive, not checking whether the \
+files were located in a subdirectory or in the top directory of the archive, \
+resulting in files scattered all over the place? aunpack overcomes this problem \
+by first extracting to a new directory. If there was only a single file in the archive, \
+that file is moved to the original directory. aunpack also prevents local files from \
+being overwritten by mistake.
+
+The other commands provided are apack (to create archives), als (to list files in \
+archives), and acat (to extract files to standard out). As atool invokes external \
+programs to handle the archives, not all commands may be supported for a \
+certain type of archives.
+
+atool identifies archives by their file extension. Sometimes this is not possible \
+- for instance rar archives usually have varying numeric file extensions. In those \
+cases when atool can't identify the format, file is used instead. (atool can be \
+configured not to use file.)"
```

Comment:
It says it can be configured to not use file, so I'm assuming it's not a hard dependency.